spec.md

```diff
@@ -4315,6 +4362,12 @@ property_expr = reference_static | property_literal_expr | property_expr_primop
 property_expr_primop = property_primop_2expr ;
 expr_primop = primop_2expr | primop_1expr | primop_1expr1int | primop_1expr2int ;
 
+expr_intrinsic = "intrinsic(" , id ,
```

Comment:
One super tiny nit:

I've been trying to separate keywords from parens in the grammar. cf:

    "release_initial" , "("

instead of 

    ""release_initial("

Reply:
Thanks, fixed!